docs/organizations/overview.mdx

```diff
@@ -101,7 +101,7 @@ You can create organizations in the Clerk Dashboard or in your application. To c
 
 #### Monthly active organization (MAO)
 
-The number of organizations you can have in a single Clerk application depends on your [Clerk plan](https://clerk.com/pricing) and the type of instance (development or production), and is measured by "monthly active organizations" (MAOs). An MAO is an organization with more than one user that has signed in that month.
+The number of organizations you can have in a single Clerk application depends on your [Clerk plan](https://clerk.com/pricing) and the type of instance (development or production), and is measured by "monthly active organizations" (MAOs). An MAO is an organization with at least two users that have signed in that month, at least one of which must be an active user.
```

Comment:
So how do we define "active" user?

Reply:
Means they've been [interacted with the organization in the last billing cycle](https://clerkinc.slack.com/archives/C05HJBXCVL4/p1716556600411229?thread_ts=1716301138.076479&cid=C05HJBXCVL4). Updating now to make it clearer 👍